bluetooth-plane/LICENSE

```diff
@@ -0,0 +1,191 @@
+Apache License
```

Comment:
There's already a copy of this license at the top level, so we probably don't need it here.